1-js/02-first-steps/13-switch/article.md

```diff
@@ -129,7 +155,7 @@ switch (a) {
   case 3:                    // (*) grouped two cases
   case 5:
     alert('Wrong!');
-    alert('Why don't you take a math class?');
+    alert("Why don't you take a math class?");
```

Comment:
because there is a single quote in the message itself, so double quotes are required, short of causing a syntax error